cmd/minishift/state/instance_dirs.go

```diff
@@ -17,42 +17,46 @@ limitations under the License.
 package state
 
 import (
-	"github.com/minishift/minishift/pkg/minikube/constants"
 	"path/filepath"
+
+	"github.com/minishift/minishift/pkg/minikube/constants"
 )
 
 type MinishiftDirs struct {
-	Home       string
-	Config     string
-	Machines   string
-	Certs      string
-	Cache      string
-	IsoCache   string
-	OcCache    string
-	ImageCache string
-	Addons     string
-	Logs       string
-	Tmp        string
+	Home         string
+	Config       string
+	GlobalConfig string
+	Machines     string
+	Certs        string
+	Cache        string
+	IsoCache     string
+	OcCache      string
+	ImageCache   string
+	Addons       string
+	Logs         string
+	Tmp          string
 }
 
 var InstanceDirs *MinishiftDirs
 
 // Constructor for MinishiftDirs
 func NewMinishiftDirs(baseDir string) *MinishiftDirs {
+	minishiftHomeDir := constants.GetMinishiftHomeDir()
 	// We use a global cache, sharing the cache directory of the default 'minishift' instance
-	cacheDir := filepath.Join(constants.GetMinishiftHomeDir(), "cache")
+	cacheDir := filepath.Join(minishiftHomeDir, "cache")
```

Comment:
Why change this, will the env variable `MINISHIFT_HOME_DIR` work if we change this?

Reply:
Sorry, didn't see you added it in the first line